forums/forms.py

```diff
@@ -34,43 +34,51 @@ def clean(self):
             raise forms.ValidationError("User is blocked")
         cleaned_data['user'] = user
         return cleaned_data
+
         
 class ProfileForm(forms.ModelForm):
     class Meta:
         model = Profile
         exclude = ['user', 'confirmation_code']
     first_name = forms.CharField(widget=forms.TextInput(),
                         required = True,
-                        error_messages = {'required':'First name field required.'})
+                        error_messages =
+                        {'required':'First name field required.'})
     last_name = forms.CharField(widget=forms.TextInput(),
                         required = True,
-                        error_messages = {'required':'Last name field required.'})
+                        error_messages =
+                        {'required':'Last name field required.'})
 
-    phone = forms.CharField(max_length = 12, widget=forms.TextInput(),required=False, validators = [RegexValidator(regex = '^[0-9-_+.]*$')])
+    phone = forms.CharField(max_length = 12,
+                            widget=forms.TextInput(),
+                            required=False,
+                            validators =
+                            [RegexValidator(regex = '^[0-9-_+.]*$')])
 
-  #   def clean_phone(self):
-  #   	phone = self.cleaned_data['phone']
-  #   	temp = str(phone)
-  #   	if not temp.isdigit():
-  #   		raise forms.ValidationError("Enter valid contact number.")
-		# return phone
+    #   def clean_phone(self):
```

Comment:
Please confirm if this code is required and remove. I would advise remove this commented code